lib/govuk_app_config/govuk_json_logging.rb

```diff
@@ -50,6 +50,11 @@ def self.configure
     # Elasticsearch index expect source to be an object and logstash defaults
     # source to be the host IP address causing logs to be dropped.
     Rails.application.config.logstasher.source = {}
+    # Elasticsearch index expect error to be an object and logstash defaults
+    # error to be a string causing logs to be dropped.
+    Rails.application.config.logstasher.field_renaming = {
+      error: :message,
```

Comment:
Did you mean this to be "error" to "error.message"? This appears to renaming the field "error" to "message".